src/lib/chips/chip-list.ts

```diff
@@ -59,11 +63,19 @@ export class MdChipList implements AfterContentInit {
   /** The chip components contained within this chip list. */
   chips: QueryList<MdChip>;
 
-  constructor(private _elementRef: ElementRef) { }
+  /** Tab index for the chip list. */
+  _tabIndex = 0;
```

Comment:
It's because the browsers determine that the previous tabbable element is the parent (since the chips are `tabindex="-1"` and the chip list is `tabindex="0"`). When the parent receives focus, it forwards it back to the first chip. As an aside, Firefox also tabs into the parent when tabbing forward.